src/components/AddTodo.js

```diff
@@ -12,18 +12,18 @@ const AddTodo = () => {
         text: todo,
         completed: false,
     }
-    setTodos ([...todos].concat(newTodo))
+    setTodos((previous) => [...previous].concat(newTodo));

```

Comment:
You are already spreading the previous values ... why not just add the new value at the last position?

Why use `.concat`?